docs/modules/ROOT/pages/index.adoc

```diff
@@ -852,179 +850,116 @@ The `validFrom` field MUST exist and it MUST express the date and time when the
+[#vc-credentialsubject-verifiedidentity-type]
+Verified identity type::
+The `verifiedIdentities[?].type` property MUST be present and MUST be a non-empty string that defines the type of verification that was performed by the _<<_identity_provider,identity provider>>._ This specification defines the following values which _<<_identity_assertion_consumer,identity assertion consumers>>_ SHOULD be prepared to accept:
++
+[width="100%",cols="4,10",options="header"]
+|=======================
+| Value        |  Meaning
+| `cawg.document_verification` | The _<<_identity_provider,identity provider>>_ has verified one or more government-issued identity documents presented by the _<<_named_actor,named actor>>._
+| `cawg.affiliation` | The _<<_identity_provider,identity provider>>_ is attesting to the _<<_named_actor,named actor’s>>_ membership in an organization. This could be a professional organization or an employment relationship.
+| `cawg.social_media` | The _<<_named_actor,named actor>>_ has demonstrated control over an account (typically a social media account) hosted by the _<<_identity_provider,identity provider>>._
+| `cawg.crypto_wallet` | The _<<_named_actor,named actor>>_ has demonstrated control over an account (typically a crypto-wallet) hosted by the _<<_identity_provider,identity provider>>._ 
+|=======================
++
+Other string values MAY be used in `verifiedIdentities[?].type` with the understanding that they may not be well understood by *<<_identity_assertion_consumer,identity assertion consumers>>.* String values for `verifiedIdentities[?].type` that begin with the prefix `cawg.` are reserved for the use of the Creator Assertions Working Group and MUST NOT be used unless defined in a this or a future version of this specification.
++
+IMPORTANT: Future minor version updates (1.1, 1.2, etc.) to this specification MAY define new values for `verifiedIdentities[?].type` using the `cawg.` prefix.
+
+[#vc-credentialsubject-verifiedidentity-name]
+Display name::
+The `verifiedIdentities[?].name` property MAY be present. If present, it MUST NOT be empty and must be a string defining the _<<_named_actor,named actor’s>>_ name as understood by the _<<_identity_provider,identity provider>>._
++
+If the `type` of this verified identity is `cawg.document_verification`, the `verifiedIdentities[?].name` property MUST be present and MUST exactly match the name found on the identity documents.
+
+[#vc-credentialsubject-verifiedidentity-username]
+User name::
+The `verifiedIdentities[?].username` property MAY be present. If present, it MUST be a non-empty text string representing the _<<_named_actor,named actor’s>>_ user name as assigned by the _<<_identity_provider,identity provider>>._
++
+If the `type` of this verified identity is `cawg.crypto_wallet`, the `verifiedIdentities[?].username` property MUST be present and MUST be the unique alphanumeric string that can be used to identity the user’s crypto wallet.
```

Comment:
for crypto wallets shouldn't we be mandated that the address property be present and for the social one the username property be present ?

Reply:
Is there a use case for distinct `address` and `username` properties? In other words, I think what we had called `address` in the previous `cryptoWallet` description seems functionally equivalent to a user name in this combined vocabulary.

I will add the requirement that user name be present for the social media case.

Reply:
It will make the data in the field more inline with the field name ... since what we'll get from the crypto wallets will be a public key address